### PR TITLE
Showing profile image when customer login.

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#5464

## Description
Showing customer profile picture if customer login and have uploaded a profile picture.
Fixed for default and velocity theme.

https://prnt.sc/oVwv235YWgj8

### CODE REVIEW

File: packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php [hunk 1]
```diff
@@ -98,7 +98,11 @@ class="search-field"
 
                 <li>
                     <span class="dropdown-toggle">
-                        <i class="icon account-icon"></i>
+                        @if( auth()->guard('customer')->user() && auth()->guard('customer')->user()->image) 
+                            <i style="vertical-align: top;"><img style="width:20px;margin-right:5px;height:20px;border-radius:50%;" src="{{ auth('customer')->user()->image_url }}" alt="{{ auth('customer')->user()->first_name }}"/></i>
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php:102]: Use SCSS for styling.

File: packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php [hunk 2]
```diff
@@ -2,8 +2,11 @@
 
 <div id="account">
     <div class="d-inline-block welcome-content dropdown-toggle">
-        <i class="material-icons align-vertical-top">perm_identity</i>
-
+        @if( auth()->guard('customer')->user() && auth()->guard('customer')->user()->image) 
+            <i class="align-vertical-top"><img style="width:20px;margin-right:5px;height:20px;border-radius:50%;" src="{{ auth('customer')->user()->image_url }}" alt="{{ auth('customer')->user()->first_name }}"/></i>
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php:6]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php (hunk 2 above)
Comment (papnoisanjeev) [reply packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php]: done.

File: packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php (hunk 1 above)
Comment (papnoisanjeev) [reply packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php]: done

File: packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php [hunk 3]
```diff
@@ -2,8 +2,11 @@
 
 <div id="account">
     <div class="d-inline-block welcome-content dropdown-toggle">
-        <i class="material-icons align-vertical-top">perm_identity</i>
-
+        @if( auth()->guard('customer')->user() && auth()->guard('customer')->user()->image)
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php:5]: Remove space after the bracket.

File: packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php [hunk 4]
```diff
@@ -2,8 +2,11 @@
 
 <div id="account">
     <div class="d-inline-block welcome-content dropdown-toggle">
-        <i class="material-icons align-vertical-top">perm_identity</i>
-
+        @if( auth()->guard('customer')->user() && auth()->guard('customer')->user()->image) 
+            <img class= "profile-small-icon" src="{{ auth('customer')->user()->image_url }}" alt="{{ auth('customer')->user()->first_name }}"/>
+        @else
+            <i class="material-icons align-vertical-top">perm_identity</i>
+        @endif
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/layouts/top-nav/login-section.blade.php:9]: Add one line here.

File: packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php [hunk 5]
```diff
@@ -98,7 +98,11 @@ class="search-field"
 
                 <li>
                     <span class="dropdown-toggle">
-                        <i class="icon account-icon"></i>
+                        @if( auth()->guard('customer')->user() && auth()->guard('customer')->user()->image)
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php:101]: Remove space after the bracket.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Review by devansh-webkul — DISMISSED** [review]

**Comment by jitendra-webkul** [comment]
Resolve conflicts

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
**not working** properly, the customer name is **not visible,** Please find the screenshot [Here](https://prnt.sc/bxRYI0Ux8nLc)
And please **resolve the conflicts.**

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
This is **working fine** **only with** the **default** theme. but **getting** **issues** with the **velocity** theme

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
The conflicts are **still not resolved.**

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 
Conflicts has been resolved here.

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
The image option will only be for the velocity theme. the uploaded image should not be visible on the default theme.

**Comment by Sanjaybhattwebkul** [comment]
And if the customer is updating the profile from the default theme, then the customer's image should not be removed.

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 

Removed profile pic for **default** theme as discussed as only need for velocity theme.

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
The profile picture should not be updated if the customer is updating his profile from the default theme.

**Comment by Sanjaybhattwebkul** [comment]
> @papnoisanjeev The profile picture should not be updated if the customer is updating his profile from the default theme.

any update?

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 

Fixed the issue (user profile remove) if anyone update profile using default theme.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
**Still not working** the customer is **not able to delete** the profile picture,

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul

Fixed.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Pull request merged**